operator-framework-core/src/main/java/io/javaoperatorsdk/operator/processing/event/source/informer/InformerEventSource.java

```diff
@@ -277,22 +280,32 @@ private void handleRecentResourceOperationAndStopEventRecording(Operation operat
       R newResource, R oldResource) {
     ResourceID resourceID = ResourceID.fromResource(newResource);
     try {
+      // what if the prev event delayed, thus that even did not clear the temp cache.
```

Comment:
I don't understand this comment

Reply:
removed that, just probably some note for myself. good catch :)